pkg/acs/calacs/lib/prinfo.c

```diff
@@ -292,15 +292,15 @@ int maxch     i: allocated length of outstr (including EOS)
    to lower case, and pad with blanks.
 */
 
-void Upper1 (char *instr, char *outstr, int maxch) {
+void Upper1 (const char *instr, char *outstr, int maxch) {
 
 /* arguments:
 char *instr   i: input string
 char *outstr  o: string with only the first letter capitalized
 int maxch     i: allocated length of outstr (including EOS)
 */
 
-	int i;
+	int i = 1;
```

Comment:
So it looks like `i` can only be un-initialized if `instr[0] == '\0'`. So, isn't it safer to just return `outstr[0] = '\0'` in that case instead of initializing `i = 1`, which does not help in that case?